src/Apps/Collect/Components/Seo/BreadCrumbList.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import { Meta } from "react-head"
+
+const { APP_URL } = process.env
+
+interface Item {
+  path: string
+  name: string
+}
+
+interface BreadCrumbListProps {
+  items: Item[]
+}
+
+export const BreadCrumbList = ({ items }: BreadCrumbListProps) => {
+  return (
+    <Meta
+      tag="script"
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{
+        __html: JSON.stringify({
+          "@context": "http://schema.org",
+          "@type": "BreadcrumbList",
+          itemListElement: [
+            {
+              "@type": "ListItem",
+              position: 1,
+              item: {
+                "@id": APP_URL,
+                name: "Artsy",
+              },
+            },
+            ...items.map(({ path, name }, index) => ({
+              "@type": "ListItem",
+              position: index + 2,
```

Comment:
I'm just curious but why do we add 2 instead of 1 to the index to calculate the position.

Reply:
Yeah this looks tricky. The example in the official doc has a list where the index starts with `1` while in programminng it starts with `0`. There's a top-level breadcrumb link for www.artsy.net as the first element, so this `+2` is important to make the `itemListElement` valid. I'll add a comment about this so the context won't be lost.